csaf_2.0/submit/README.md

```diff
@@ -0,0 +1,84 @@
+
+a. *The name(s) of the submission requester(s), that is, the TC, the Member Section, or the OASIS*
+   *Organizational Members that support the submission request as described in section 1(d).*
+
+OASIS Common Security Advisory Framework (CSAF) TC
+
+b. *The name of the intended receiving standards organization. The request may also suggest the committee*
+   *or group in that organization which should process that submission.*
+
+ISO/IEC\ JTC\ 1 Secretariat (suggesting “SC\ 27 Information security, cybersecurity and privacy
+protection” as the published scope at https://www.iso.org/committee/45306.html indicates)
+
+c. *The intended status or outcome that the request seeks from the receiving organization’s process; and a*
+   *short description of the receiving organization’s approval process, including estimated time required,*
+   *stages of approval and who votes at each stage.*
+
+Intended status: Advance CSAF\ v2.0 OASIS Standard to an ISO/IEC International Standard
+
+Process: ISO/IEC\ JTC\ 1 PAS Transposition Process involving a Draft International
+Standard\(DIS) ballot followed by a Ballot Resolution Meeting\ (BRM) (if there are comments
+during ballots), and potentially a Final DIS\ (FDIS) Ballot.
+
+Stages of approval: https://www.iso.org/stage-codes.html - Stage 40
+
+
+*Note: Submission request continued on next page*
+
+                                              *... continued from first page of submission request.*
+
+d. *An explanation of how the submission will benefit OASIS.*
+
+Advancing CSAF\ v2.0 to an international standard draws international attention to the work
+done by OASIS and to the organization itself,and governments/regulators may be more
+inclined to use the standard because of its ISO/IEC status.
+
+This will help nurture and expand the ongoing liaisons with ISO/IEC JTC 1.
+
+Further, this submission will help maintain OASIS in good standing as a recognized ISO/IEC
+JTC\ 1 PAS submitter organization.
+
+Last, but not least, as ISO/IEC standard, the process requirements for the success of the
+CSAF\ v2.0 OASIS Standard will be more easily enforceable through certification and audit.
+
+Security advisories still require essential inputs from human experts within organizations and
+rely on organizational processes that plausibly implement the preconditions for successful
+implementation.
+
+e. *The expected licensing, copyright and other intellectual property terms that will be used by the receiving*
+   *organization in regard to the submission.*
+
+ISO/IEC JTC\ 1 will expect that OASIS will abide by the ISO/IEC/ITU-T Common Patent Policy
+(indicated by at least notifying ISO/IEC of any patent declarations in the required Explanatory
+Report), and will provide ISO/IEC with sufficient copyright license to modify and publish the
+resulting ISO/IEC Standard.
+
+As these items have already been agreed with ISO/IEC JTC\ 1 when OASIS was approved as a
+PAS Submitter, no new issues are expected.
+
```

Comment:
Is there also a need here to mention, that the CSAF v2.0 should be provided by ISO for download free of charge?
Flagging @chet-ensign here as well.

Reply:
I do not see that need for adding to the request, as OData did not mention and it is free of charge. It may not even be the place as we simply satisfy the liaison policy requirements on a written submission.